ReSharper.FSharp/src/FSharp.Psi/src/Impl/DeclaredElement/FSharpMethodParameter.cs

```diff
@@ -55,7 +55,7 @@ public override DefaultValue GetDefaultValue()
     }
 
     public override ParameterKind Kind => FSharpSymbol.MapParameterKind();
-    public override bool IsParameterArray => Owner is IFSharpMethod && FSharpSymbol.IsParamArrayArg;
+    public override bool IsParameterArray => Owner is IFSharpFunction && FSharpSymbol.IsParamArrayArg;
```

Comment:
Possibly we should rename FSharpMethodParameter to FSharpParameter?

Reply:
Yes, I think we could do that.

Reply:
Though, it would have been better if it was a separate commit, as the fix for the param array is a bit hidden by unrelated changes now.